apps/mobile-wallet/package.json

```diff
@@ -48,7 +48,7 @@
     "big-integer": "^1.6.51",
     "colord": "^2.9.3",
```

Comment:
Some headers are wrong:
1. Import wallet screen
2. Addresses screen

<img width="350" alt="image" src="https://github.com/user-attachments/assets/b2f72413-bdc1-4b4b-9472-099f25a6aa11">


<img width="350" alt="image" src="https://github.com/user-attachments/assets/c1280bb2-53c7-4b76-a8ec-d390e6eb4c9f">

Reply:
First is fixed in this PR: https://github.com/alephium/alephium-frontend/pull/1032
Second is already merged. 

Resolving.